.gitignore

```diff
@@ -1,6 +1,12 @@
 Library/
 Temp/
+.vs/
+UnityPackageManager/
+	
 
 .DS_Store
 Thumbs.db
-*.sln
\ No newline at end of file
+*.sln
+unitydemo-2DPlatformer.csproj
+
+*.unitydemo-2DPlatformer.csproj
```

Comment:
Something that we probably should have done here is to add the items from here.

https://github.com/github/gitignore/blob/master/Unity.gitignore

Reply:
Indeed, it's a much cleaner version than mine.
Since I'm going to make a new commit. Let me to add this version.